modules/oe_theme_content_event/src/Plugin/ExtraField/Display/InfoDisclosureExtraFieldBase.php

```diff
@@ -76,22 +76,13 @@ protected function isCurrentDay(int $timestamp): bool {
   }
 
```

Comment:
Can we improve the method description for isCurrentDay? Maybe something like: Asserts if a given timestamp corresponds to the same day of the current request time.